### ISSUE TITLE
Degree tuition tab links - deactivated tab needs explicit aria-selected="false"

### ISSUE THREAD

**Issue by cjg89** [UCF/Online-Child-Theme#96]
**Describe the bug**

The single degree tuition tabs' inactive link currently prints an empty `aria-selected` value, which is getting flagged as invalid (it must explicitly be "false", not "").
https://github.com/UCF/Online-Child-Theme/blob/master/includes/degree-functions.php#L133

**Steps to Reproduce**

1. Navigate to a single degree
2. Inspect the out-of-state tuition tab `<a>`
3. Confirm the `aria-selected` attribute value is `""`

**Expected Behavior**

`aria-selected` should be `"true"` or `"false"` on the out-of-state tab on page load, depending on whether or not in-state tuition information is available.

**Possible Fix**

Update the line shown above to explicitly print "true" or "false"; e.g.
```php
aria-selected="<?php echo ( $resident_tuition ) ? 'false' : 'true'; ?>"
```

**Your environment:**

n/a

**Additional context**

Flagged by SiteImprove.
